src/functions/exportFunctions.js

```diff
@@ -121,7 +125,47 @@ export const fetchAndTransformQuizzes = async (
   }
 };
 
-export async function fetchData(
+export const fetchAndAddUserQuizzes = async (
+  backendApiCall,
+  quizzes,
+  onSuccess,
+  onError,
+  setUserQuizzesUpdated,
+  auth
+) => {
+  if (!auth.loggedIn) return;
+
+  try {
+    const userQuizDataArray = await backendApiCall(
+      'GET',
+      `/api/v1/progress/user?userId=${auth.userId}`
+    );
+
+    const updatedQuizzes = quizzes.map((quiz) => {
+      const userQuizData = userQuizDataArray.data.find(
+        (uqd) => uqd.test === quiz.id
+      );
+      if (userQuizData) {
+        return {
+          ...quiz,
+          quizProgress: {
+            attemptsCount: userQuizData.attemptNumber.toString(),
```

Comment:
Similar to my comment about color contrast on the previous PR, the Attempts text is kind of hard to read since it's a light gray color on a white background. What do you think about either darkening the text, bolding the text, or making the attempts text larger to make it easier to read?

Reply:
Done! Again, I'll defer to our design guru Kat on final implementation, but I have darkened the color and increased the fontWeight.